src/components/SignUpForm.js

```diff
@@ -12,377 +12,391 @@ import Datetime from "elevate-ui/Datetime";
 import moment from "moment";
 
 class SignUpForm extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      formState: null,
-    };
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            formState: null,
+        };
+    }
+
+    componentDidMount() {
+        const url = 'http://localhost:8005/api/marketing-site/ip';
```

Comment:
Here we need to have some sort of an environment based url

Reply:
this const should be a little more descriptive